metrics/metrics-integration/metrics-prometheus/src/main/java/org/apache/servicecomb/metrics/prometheus/MetricsHttpPublisher.java

```diff
@@ -37,20 +36,34 @@
 public class MetricsHttpPublisher implements ApplicationListener<ApplicationEvent> {
   private static final Logger LOGGER = LoggerFactory.getLogger(MetricsHttpPublisher.class);
 
-  private static final String METRICS_PROMETHEUS_PORT = "servicecomb.metrics.prometheus.port";
+  private static final String METRICS_PROMETHEUS_ADDRESS = "servicecomb.metrics.prometheus.address";
```

Comment:
Using the host and port setting could be a better way to go.

Reply:
may better keep same as "cse.rest.adderss" ? or I will do as  your suggestion